VocaDbWeb/Startup.cs

```diff
@@ -370,6 +370,8 @@ void HandleHttpError(int code, string? description = null, string? msg = null)
 				endpoints.MapControllerRoute(
 					name: "default",
 					pattern: "{controller=Home}/{action=Index}/{id?}");
+
+				endpoints.MapFallbackToController(action: "Index", controller: "React");

```

Comment:
The `ReactController` controller will handle requests to the React app, which will simply serve the single page in the React app.